SeatsioDotNet.Test/Events/UpdateEventTest.cs

```diff
@@ -142,5 +142,20 @@ public void UpdateDate()
             var retrievedEvent = Client.Events.Retrieve(evnt.Key);
             Assert.Equal(new DateOnly(2022, 1, 10), retrievedEvent.Date);
         }
+        
+        [Fact]
+        public void UpdateIsInThePast()
+        {
+            var chartKey = CreateTestChart();
+            Client.Seasons.Create(chartKey, eventKeys: new[] {"event1"});
+
+            Client.Events.Update("event1", new UpdateEventParams().WithIsInThePast(true));
+            var retrievedEvent1 = Client.Events.Retrieve("event1");
+            Assert.True(retrievedEvent1.IsInThePast);
+
+            var e = Assert.Throws<SeatsioException>(() =>
```

Comment:
I wouldn't test events in the past being immutable - that's covered by the tests in `seatsio-core-v2`.